elephant/spike_train_generation.py

```diff
@@ -316,33 +321,89 @@ def homogeneous_poisson_process(rate, t_start=0.0 * ms, t_stop=1000.0 * ms,
 
     rate : Quantity scalar with dimension 1/time
            The rate of the discharge.
-    t_start : Quantity scalar with dimension time
+    t_start : Quantity scalar with dimension time, optional
               The beginning of the spike train.
-    t_stop : Quantity scalar with dimension time
+              Default: 0 ms
+    t_stop : Quantity scalar with dimension time, optional
              The end of the spike train.
-    as_array : bool
+             Default: 1000 ms
+    as_array : bool, optional
                If True, a NumPy array of sorted spikes is returned,
                rather than a SpikeTrain object.
+               Default: False
+    refractory_period : None or Quantity, optional
```

Comment:
it should be `pq.Quantity` for all arguments, actually. Refer to https://github.com/NeuralEnsemble/elephant/blob/master/elephant/unitary_event_analysis.py#L679 for example